src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -71,15 +75,19 @@ export const Order2PaymentForm: React.FC<Order2PaymentFormProps> = ({
 
   return (
     <Elements stripe={stripePromise} options={options}>
-      <PaymentFormContent setConfirmationToken={setConfirmationToken} />
+      <PaymentFormContent
+        order={order}
+        setConfirmationToken={setConfirmationToken}
+      />
     </Elements>
   )
 }
 
-const PaymentFormContent = ({ setConfirmationToken }) => {
+const PaymentFormContent = ({ order, setConfirmationToken }) => {
```

Comment:
non-blocking: it would be good to get real types in here. non-blocking but we should do it as a quick followup.